code/modules/reagents/Chemistry-Machinery.dm

```diff
@@ -1903,9 +1904,13 @@ USE THIS CHEMISTRY DISPENSER FOR MAPS SO THEY START AT 100 ENERGY
 		detach()
 		return
 
+/obj/structure/centrifuge/AltClick()

```

Comment:
I plan to add more of them, to the point where it becomes intuitive to alt-click everything with a verb to see if it works